pkg/corerp/datamodel/environment.go

```diff
@@ -38,11 +38,42 @@ func (e *Environment) ResourceTypeName() string {
 
 // EnvironmentProperties represents the properties of Environment.
 type EnvironmentProperties struct {
-	Compute    rpv1.EnvironmentCompute                           `json:"compute,omitempty"`
-	Recipes    map[string]map[string]EnvironmentRecipeProperties `json:"recipes,omitempty"`
-	Providers  Providers                                         `json:"providers,omitempty"`
-	Extensions []Extension                                       `json:"extensions,omitempty"`
-	Simulated  bool                                              `json:"simulated,omitempty"`
+	Compute      rpv1.EnvironmentCompute                           `json:"compute,omitempty"`
+	Recipes      map[string]map[string]EnvironmentRecipeProperties `json:"recipes,omitempty"`
+	Providers    Providers                                         `json:"providers,omitempty"`
+	RecipeConfig RecipeConfigProperties                            `json:"recipeConfig,omitempty"`
+	Extensions   []Extension                                       `json:"extensions,omitempty"`
+	Simulated    bool                                              `json:"simulated,omitempty"`
+}
+
+// RecipeConfigProperties - Specifies recipe configurations needed for the recipes.
+type RecipeConfigProperties struct {
+	// Specifies the terraform config properties
```

Comment:
We should start with the name of the property. All docs should follow that structure.

Reply:
done